ios/GigyaSdkWrapper.swift

```diff
@@ -99,6 +111,19 @@ class GigyaSdkWrapper<T: GigyaAccountProtocol>: GigyaSdkWrapperProtocol {
         return gigya.isLoggedIn()
     }
 
+    func getSession() {
+        let mapped = self.sessionToDic(session: gigya.getSession())
+        self.promise?.resolve(result: mapped)
+    }
+
+    func setSession(token: String, secret: String, expiration: NSNumber = 0) {
+        let session: GigyaSession! = GigyaSession(sessionToken: token, secret: secret, expiration: expiration as? Double)
+        gigya.setSession(session)
+        
+        let mapped = self.sessionToDic(session: session)
```

Comment:
Instead of adding a new function to create a dictionary from the session object just do:

`let sessionAsMap = [“token”: session?.token, “secret”: session?.secret, “sessionExpirationTimestamp”: session.session?.sessionExpirationTimestamp]
`

No need for a conversion function.

Reply:
I still need the conversion function since I'm using it in two different places.